strongbox-data-service/pom.xml

```diff
@@ -79,7 +79,7 @@
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>strongbox-db-liquibase</artifactId>
-            <version>${project.version}</version>
+            <version>PR-5-1.0-SNAPSHOT</version>
```

Comment:
Usually, the accepted (industry standard) Maven convention for this is `${baseVersion}-${issueKey}-SNAPSHOT`. For example: `1.0-SB-1234-SNAPSHOT`.

This is just for future reference and it doesn't make sense to change it right now.